libkbfs/folder_update_prepper.go

```diff
@@ -476,7 +480,7 @@ func (fup folderUpdatePrepper) prepTree(ctx context.Context, lState *lockState,
 // updateResolutionUsage figures out how many bytes are referenced and
 // unreferenced in the merged branch by this resolution.  Only needs
 // to be called for non-squash resolutions.
-func (fup folderUpdatePrepper) updateResolutionUsage(ctx context.Context,
+func (fup folderUpdatePrepper) updateResolutionUsageLocked(ctx context.Context,
```

Comment:
This seems to imply that the `lState` should be locked, but it actually refers to the `cacheLock`. Should the name reflect that? Same thing below.